web/index.html

```diff
@@ -70,287 +68,350 @@ <h3 class="sub-header">
+                                                            @click="showHiddenRecipes = false">
+                                                            {{currentHiddenToLength}} entries shown
+                                                        </div>
+                                                    </div>
+                                                </div>
+                                            </transition>
+                                            <div v-else style="font-weight: bold; padding-top: 5px;">
+                                                No recipes found
+                                            </div>
                                         </div>
-                                    </div>
+                                    </transition>
                                     <div id="element-scroll-end"></div>
                                 </div>
                             </div>
                         </div>
                     </div>
-                    <transition name="slide-up" mode="out-in">
-                        <div :key=0 class="nav-bar">
-                            <div class="nav-bar-link" :class="{ 'selected-link': currentPanelIndex === -1 }"
-                                @click="currentPanelIndex = -1">
-                                Created By
-                            </div>
-                            <div class="nav-bar-link" :class="{ 'selected-link': currentPanelIndex === 0 }"
-                                @click="currentPanelIndex = 0">
-                                Crafting Recipe
-                            </div>
-                            <div class="nav-bar-link" :class="{ 'selected-link': currentPanelIndex === 1 }"
-                                @click="currentPanelIndex = 1">
-                                Used In
-                            </div>
-                        </div>
-                    </transition>
                 </div>
                 <div key=1 v-else>
                     <div style="display: flex; flex-wrap: wrap; justify-content: center;">
                         <div v-if="Object.keys(data.index).length == 0" style="font-weight: bold; padding-top: 5px;">
                             Loading data...</div>
-                        <div class="item" :class="{'item-used': !saveData.includes(element)}"
-                            v-for="element in elementList" @click="navigateTo(element)">
+                        <div class="item" :class="{'item-used': !saveData.has(element)}" v-for="element in elementList"
+                            @click="navigateTo(element)" @mouseover="loadChunk(getChunk(element))">
                             {{ data.index[element][0] }}&nbsp;{{ data.index[element][1] }}
                         </div>
                     </div>
                     <div id="scroll-end"></div>
                 </div>
             </transition>
+            <div v-if="currentElement" class="nav-bar">
```

Comment:
Moved the bottom bar down here so that it wouldn't be affected by transitions, which was causing it to be glitched in appearance in some cases while things were loading.